components/Home.js

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import { Appbar, IconButton, Text, Subheading, Menu } from 'react-native-paper'
+import { useAuthenticationContext } from '../context/authentication'
+
+import theme from '../default-theme'
+
+const UI_STRINGS = {
+  default: {
+    heading: `You don't have a secret set up yet.`,
+    description:
+      'Add one by scanning or uploading a QR code, or even enter the details manually.',
+  },
+}
+
+const A11Y_LABELS = {
+  code: 'Add new secret by scanning a QR Code',
+  upload: 'Add new secret by uploading a QR Code',
+  add: 'Add new secret by filling the details',
+}
+
+export default function Home() {
+  const [isMenuActive, setMenuActive] = React.useState(false)
+  const { handleLogout } = useAuthenticationContext()
+  return (
+    <View style={styles.container}>
+      <Appbar style={styles.bottom}>
+        <Appbar.Content title={<Text style={styles.title}>Optic</Text>} />
+        <View>
+          <Menu
+            visible={isMenuActive}
+            onDismiss={() => setMenuActive(false)}
+            anchor={
+              <Appbar.Action
+                icon="account"
+                onPress={() => setMenuActive(true)}
+                color="white"
+              />
+            }
+          >
+            <Menu.Item onPress={() => handleLogout()} title="Logout" />
+          </Menu>
+        </View>
+      </Appbar>
+      <View style={styles.description}>
+        <Subheading style={styles.descriptionText}>
+          {UI_STRINGS.default.heading}
+        </Subheading>
+        <Subheading style={styles.descriptionText}>
+          {UI_STRINGS.default.description}
+        </Subheading>
+      </View>
+      <View style={styles.actions}>
+        <IconButton
+          icon="qrcode"
+          accessibilityLabel={A11Y_LABELS.code}
+          mode="contained"
+          style={styles.button}
+          size={40}
+          color={theme.colors.white}
+        />
+        <IconButton
+          icon="upload"
+          accessibilityLabel={A11Y_LABELS.upload}
+          mode="contained"
+          style={styles.button}
+          size={40}
+          color={theme.colors.white}
+        />
+        <IconButton
+          icon="import"
+          accessibilityLabel={A11Y_LABELS.add}
+          mode="contained"
+          style={styles.button}
+          size={40}
+          color={theme.colors.white}
+        />
+      </View>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexGrow: 1,
+    paddingTop: theme.spacing.small,
+    width: '100%',
+  },
+  actions: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+  },
+  title: {
+    color: theme.colors.white,
+    fontSize: theme.text.small,
+    fontWeight: '700',
```

Comment:
same comment as before. let's use typography components so we never have to specify font styles in stylesheets